packages/apps/fortune/recording-oracle/src/common/guards/signature.auth.ts

```diff
@@ -42,11 +38,10 @@ export class SignatureAuthGuard implements CanActivate {
       if (isVerified) {
         return true;
       }
-    } catch (error) {
-      // eslint-disable-next-line no-console
-      console.error(error);
+    } catch (error: any) {
+      throw new ValidationError(error.message);
```

Comment:
Probably it shouldn't be 400: imagine there is a network error while getting escrow data, it's 500, not 400 then
Would be better to log (with common logger) and re-throw or even remove try-catch at all